src/main/java/eu/arrowhead/blacklist/service/ConfigService.java

```diff
@@ -0,0 +1,60 @@
+package eu.arrowhead.blacklist.service;
```

Comment:
ConfigService is moved to the common repo. You can use that, this is not needed anymore. Sorry. Actually, when we realize that both blacklist and authentication systems (and others in the future) need  these, we decided to refactor the log and config operations into a separate service.